src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -926,7 +1034,7 @@ object Desugar {
                 for {
                   re <- goE(r)
                   dL <- leftOfAssignmentD(l)(re.typ)
-                  le = in.Assignee.Var(dL)
+                  le = dL
```

Comment:
why declare `le`? It appears useless here

Reply:
I added that to help with debugging (the debugger wouldn't show me the value otherwise), and I forgot to remove if after. Fixed